index.bs

```diff
@@ -130,21 +130,34 @@ handle additional, application-specific use cases.
     SecureContext
   ] interface Sanitizer {
     constructor(optional SanitizerConfig config = {});
+
     DocumentFragment sanitize(SanitizerInput input);
     DOMString sanitizeToString(SanitizerInput input);
+
+    SanitizerConfig config();
+    static SanitizerConfig defaultConfig();
```

Comment:
I'm surprised this is a method and not a property. Why?

Reply:
https://heycam.github.io/webidl/#idl-dictionaries says: "Dictionaries must not be used as the type of an attribute or constant," and SanitizerConfig is a dictionary.

I originally had it as an attribute, too, and then changed it when I stumbled on that sentence. If we can keep it as an attribute, I'd be happy to.  I don't understand _why_ WebIDL says this, unfortunately.

Reply:
I was hoping to find an answer for the idl question by looking at their revision history, but https://github.com/heycam/webidl/commit/8d4e7ca67bcb4de476aab0411531dc97f6d17e0d didn't help me.

Either way, let's keep it as a method :)

Reply:
We should not degrade API usability due to Web IDL issues. Web IDL exists to serve the Web Platform, not the other way around. [I just opened an issue](https://github.com/heycam/webidl/issues/981) in the Web IDL repo to ask about this.

Reply:
You could use a getter.

> It might well be that sanitizer would be better off exposing these as getters though, that's how we often expose such settings (e.g., on `ShadowRoot`).